datadog/resource_datadog_dashboard.go

```diff
@@ -5953,35 +5953,64 @@ func getWidgetCustomLinkSchema() map[string]*schema.Schema {
 		"label": {
 			Description: "The label for the custom link URL.",
 			Type:        schema.TypeString,
-			Required:    true,
+			Optional:    true,
 		},
 		"link": {
 			Description: "The URL of the custom link.",
 			Type:        schema.TypeString,
-			Required:    true,
+			Optional:    true,
+		},
+		"is_hidden": {
+			Description: "The flag for toggling context menu link visibility",
+			Type:        schema.TypeBool,
+			Optional:    true,
+		},
+		"override_label": {
+			Description: "The label id that refers to a context menu link item",
```

Comment:
```suggestion
			Description: "The label id that refers to a context menu link item.",
```

Reply:
Could we also document that `label` should not be set if `override_label` is set to true